app/views/search/_filters.html.erb

```diff
@@ -25,16 +25,16 @@
   <% if user_signed_in? && defined? @category %>
     <%= label_tag :save_as_default, 'Is default for this category?' %>
     <%= check_box_tag :save_as_default, @category.id, false, { class: 'filter-is-default form-checkbox-element' } %>
-  <% end %>
+    <% end %>
```

Comment:
Accidental dedent?

Reply:
Looks like all the other `<% end %>` lines are aligned with the statements in the block and this was the outlier.  I didn't notice this change, but I think, based on the rest of the file, it was inconsistent before.  (I don't care, so if someone wants to speak authoritatively on our coding standards, just let me know what to do.)